### ISSUE TITLE
Closed Visualizer loads the CPU

### ISSUE THREAD

**Issue by igorkovalchuk** [begoon/rk86-js#8]
Visualizer нагружает CPU даже если его закрыть.

https://youtu.be/zxd3xFRMjU8

Сначала я использую vokoscreen, поэтому потребление CPU растет,
но как только я запускаю Visualizer оно существенно возрастает,
и если я закрою Visualizer - это не помогает, и видно, что demo тоже замедлено,
нужно закрывать вкладку Chrome.
Я не использую Visualizer, но случайно заметил это.

Chrome 89.0.4389.90 64-bit, Linux - Debian 9

**Comment by begoon** [comment]
Да, пока этот так. Для полного выключения вижуалайзера надо полность перезагрузить страницу эмулятора.

https://rk86.ru/help.html#visualizer
